packages/cli/src/commands/config/set.ts

```diff
@@ -11,27 +13,26 @@ export default class Set extends BaseCommand {
     },
   }
 
-  static examples = [
-    'set --node ws://localhost:2500',
-    'set --node <geth-location>/geth.ipc',
-    'set --gasCurrency 0x874069Fa1Eb16D44d622F2e0Ca25eeA172369bC1',
-  ]
+  static examples = ['set --node ws://localhost:2500', 'set --node <geth-location>/geth.ipc']
```

Comment:
nit / cool idea example for `set --node mainnet`

Reply:
yes!